index.bs

```diff
@@ -1000,272 +984,131 @@ that user agents must support on {{FileReader}} as DOM attributes:
 
 ### FileReader States ### {#blobreader-state}
 
+<div class="note domintro">
 The {{FileReader}} object can be in one of 3 states.
-The <dfn attribute for=FileReader id="dfn-readyState">readyState</dfn> attribute,
-on getting,
-must return the current state,
-which must be one of the following values:
-
-<dl dfn-type=const dfn-for=FileReader>
-  <dt><dfn id="dfn-empty">EMPTY</dfn> (numeric value 0)
-  <dd>The {{FileReader}} object has been constructed,
-    and there are no pending reads.
-    None of the <a>read methods</a> have been called.
-    This is the default state of a newly minted {{FileReader}} object,
-    until one of the <a>read methods</a> have been called on it.
-  <dt><dfn id="dfn-loading">LOADING</dfn> (numeric value 1)
-  <dd>A {{File}} or {{Blob}} is being read.
-    One of the <a>read methods</a> is being processed,
-    and no error has occurred during the read.
-  <dt><dfn id="dfn-done">DONE</dfn> (numeric value 2)
-  <dd>The entire {{File}} or {{Blob}} has been read into memory,
-    OR a <a>file read error</a> occurred,
-    OR the read was aborted using {{FileReader/abort()}}.
-    The {{FileReader}} is no longer reading a {{File}} or {{Blob}}.
-    If {{FileReader/readyState}} is set to {{FileReader/DONE}}
-    it means at least one of the <a>read methods</a> have been called on this {{FileReader}}.
-</dl>
+The {{FileReader/readyState}} attribute tells you in which state the object is:
+
+: {{EMPTY}} (numeric value 0)
```

Comment:
Personally I like having the dfns here and the IDL link to them. For constants it could go either way, but right now the readyState getter is not defined anywhere. (Related to the internal slots thing.)